### ISSUE TITLE
Very low accuracy with torch vision mobilenet_v2 ptq model

### ISSUE THREAD

**Issue by soyebn** [intel/neural-compressor#84]
I am using the following command at, 
.../neural-compressor/examples/pytorch/image_recognition/torchvision_models/quantization/ptq/cpu/ipex/
python main.py -t --ipex -a mobilenet_v2 --pretrained /data/ssd/datasets/ilsvrc/2012/data-folders/val/

But I am getting almost 0% accuracy with quantization. If I use -a resnet18, I get quantization accuracy close to float, so setup and other dataset related paths seems to be fine.
Do I have to change anything else for mobilenet_v2?
